client/components/HistoryTotalOodlet.vue

```diff
@@ -30,13 +30,34 @@
 <script>
   import moment from 'moment';
   
-  export default{
+  export default {
+    data() {
+      return {
+        offices: []
+      }
+    },
+    
     props: ['totalOodlet'],
     
     computed: {
       orderedOn() {
         return moment(this.totalOodlet.orderedOn).locale('hr').format('LL');
       }
+    },
+    
+    methods: {
+      getOffices(){
```

Comment:
Although, my pre-historic original suggestion was to save entire oodlets in totalOodlet, which would render this obsolete. The consensus was that it required a lot of work to enable reactivity, and we're not going to do it because of time constraints. 

Hence the
 > Just a comment, no need to change it now.

Just want to make you feel as bad as me about a part of code :kiss: